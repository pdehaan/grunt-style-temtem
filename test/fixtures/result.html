<style>

</style>

<body>
<div class="box">

<!-- /template container BGN -->
<div class="template mrm">

		<!-- template start/ -->
		<div class="sample-stage">
			
<p class="attr-icon1">アイコン</p>
<p class="attr-icon2">アイコン</p>
<p class="attr-icon3">アイコン</p>			

		</div>
		<!-- template end/ -->

		<!-- template start/ -->
		<div class="sample-stage">
			
<ul>
	<li>Step1 text</li>
	<li class="next-down">▼</li>
	<li>Step2 text</li>
	<li class="next-down">▼</li>
	<li>Step3 text</li>
	<li class="next-down">▼</li>
	<li>Step4 text</li>
</ul>
<p class="next-down">▼</p>

		</div>
		<!-- template end/ -->


	[% INCLUDE 'sp/footer.inc' %]
	[% INCLUDE 'sp/menu.inc' %]
	[% INCLUDE 'sp/footer_mobage.inc' %]
</div>
<!-- template container END/ -->


<!-- /code container BGN -->
<div class="flex">

	<!-- codes start/ -->
	<div class="codes">
		<h1>mixin-attr-icon</h1>

<ul>
<li>@include mixin-attr-icon(1);</li>
<li>@include mixin-attr-icon(2);</li>
<li>@include mixin-attr-icon(3);</li>
</ul>
		<textarea>

<p class="attr-icon1">アイコン</p>
<p class="attr-icon2">アイコン</p>
<p class="attr-icon3">アイコン</p>			

		</textarea>
	</div>
	<!-- codes end/ -->

	<!-- codes start/ -->
	<div class="codes">
		<h1>mixin-flow-next-down</h1>
		<textarea>

<ul>
	<li>Step1 text</li>
	<li class="next-down">▼</li>
	<li>Step2 text</li>
	<li class="next-down">▼</li>
	<li>Step3 text</li>
	<li class="next-down">▼</li>
	<li>Step4 text</li>
</ul>
<p class="next-down">▼</p>

		</textarea>
	</div>
	<!-- codes end/ -->

</div>
<!-- code container END/ -->

</div>
</body>
</html>

