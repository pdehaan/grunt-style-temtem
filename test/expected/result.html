<style>
body {
  background: #fafafa; }

/*
# mixin-attr-icon
- @include mixin-attr-icon(1);
- @include mixin-attr-icon(2);
- @include mixin-attr-icon(3);

```
<p class="attr-icon1">アイコン</p>
<p class="attr-icon2">アイコン</p>
<p class="attr-icon3">アイコン</p>			
```
@include mixin-attr-icon(1);
@include mixin-attr-icon(2);
@include mixin-attr-icon(3);
*/
/*
# mixin-next-down

```
<ul>
	<li>Step1 text</li>
	<li class="next-down">▼</li>
	<li>Step2 text</li>
	<li class="next-down">▼</li>
	<li>Step3 text</li>
	<li class="next-down">▼</li>
	<li>Step4 text</li>
</ul>
<p class="next-down">▼</p>
```
@include mixin-next-down();
*/
.attr-icon1 {
  position: absolute;
  top: 0;
  left: 0; }

.attr-icon2 {
  position: absolute;
  top: 0;
  left: 0; }

.attr-icon3 {
  position: absolute;
  top: 0;
  left: 0; }

.next-down {
  margin: 0;
  padding: 0; }

</style>

<body>
<div class="box">

<!-- /template container BGN -->
<div class="template mrm">

		<!-- template start/ -->
		<div class="sample-stage">
			
<p class="attr-icon1">アイコン</p>
<p class="attr-icon2">アイコン</p>
<p class="attr-icon3">アイコン</p>			

		</div>
		<!-- template end/ -->

		<!-- template start/ -->
		<div class="sample-stage">
			
<ul>
	<li>Step1 text</li>
	<li class="next-down">▼</li>
	<li>Step2 text</li>
	<li class="next-down">▼</li>
	<li>Step3 text</li>
	<li class="next-down">▼</li>
	<li>Step4 text</li>
</ul>
<p class="next-down">▼</p>

		</div>
		<!-- template end/ -->

</div>
<!-- template container END/ -->


<!-- /code container BGN -->
<div class="flex">

	<!-- codes start/ -->
	<div class="codes">
		<h1>mixin-attr-icon</h1>

<ul>
<li>@include mixin-attr-icon(1);</li>
<li>@include mixin-attr-icon(2);</li>
<li>@include mixin-attr-icon(3);</li>
</ul>
		<textarea>

<p class="attr-icon1">アイコン</p>
<p class="attr-icon2">アイコン</p>
<p class="attr-icon3">アイコン</p>			

		</textarea>
	</div>
	<!-- codes end/ -->

	<!-- codes start/ -->
	<div class="codes">
		<h1>mixin-next-down</h1>
		<textarea>

<ul>
	<li>Step1 text</li>
	<li class="next-down">▼</li>
	<li>Step2 text</li>
	<li class="next-down">▼</li>
	<li>Step3 text</li>
	<li class="next-down">▼</li>
	<li>Step4 text</li>
</ul>
<p class="next-down">▼</p>

		</textarea>
	</div>
	<!-- codes end/ -->

</div>
<!-- code container END/ -->

</div>
</body>
</html>

